api/python/cellxgene_census/src/cellxgene_census/_release_directory.py

```diff
@@ -35,9 +40,31 @@
 )
 CensusDirectory = Dict[CensusVersionName, Union[CensusVersionName, CensusVersionDescription]]
 
+CensusMirrorName = str  # name of the mirror
+CensusMirror = TypedDict(
+    "CensusMirror",
+    {
+        "provider": str,  # provider of the mirror. Only S3 is supported in this version.
+        "base_uri": str,  # name of the bucket or resource
```

Comment:
comment is misleading.  This is the "Base (absolete) URI of bucket or resource containing the Census mirror", not a "name"